### DISCUSSION TITLE
Vinculo de contatos não aparece no aplicativo

### DISCUSSION THREAD

**Discussion by andersonvass** [rictom/rede-cnpj#21] [General]
Bom dia.

Ao pesquisar pelo cnpj, não está aparecendo a ligação com os dados de contato (exemplo telefone). Quando pesquiso por um CNPJ (exemplo: 00995043000154) ele traz apenas a ligação com os sócios. Na versão disponibilizada na web, trazia o telefone como um nó vinculado aos outros nós.

Para habilitar isso é algo nas configurações ou é algum script a ser executado?

No Aguardo

Até mais

**Comment by rictom** [comment]
para aparecer esses vínculos, vc tem que ter gerado o arquivo cnpj_links_ete.db, usando o script que está nas orientações. Se você já tem esse arquivo, tem que mexer lá no rede.ini e achar a linha correspondente e colocar
base_endereco_normalizado = cnpj_links_ete.db
